content/docs/guide/gamemodes/climb.md

```diff
@@ -8,19 +8,42 @@ tags:
-
-Climb maps have evolved heavily since Kreedz made them. They became less focused on self-improvement, and are now typically setup in a race format where it's easy to compete with friends and foes alike to get the fastest times possible.
+Climb started in Counter-strike 1.1. A mapper named Kreedz decided to make a fun mess-around map that did not have the usual defuse objective for his friends. Kreedz enjoys climbing as a hobby, so he felt that there should be some trees you could climb. Kreedz found this aspect of the map to be particularly fun. He made more maps with a focus on this climbing aspect, and always named them with a kz_ tag as a sort of signature. Eventually they caught on and the kz_ tag stuck! It has become so popular that it has branched off into other games like **Half-Life** and even spawned its own game, [Kreedz Climbing](https://store.steampowered.com/app/626680/Kreedz_Climbing/). [Xtreme-Jumps](https://xtreme-jumps.eu/news.php) is currently the central hub for CS 1.6 Climb
 
 ## How Climb Works
 
-Climb works in the same way that bunny hopping does; pressing W or S combined with a strafe key causes the player to receive more velocity than simply holding W or S. Combining this with turning the player's camera in line with the strafe key, a player can get lots of momentum to jump to far platforms. This is normally called a **prestrafe**.
+### Strafing
+Strafing is the most common focus for maps and what sets Climb apart from the Bhop and surf game modes. Strafing can come in the form of long jumps and taking indirect lines. 
+Strafing works in the same way that bunny hopping does; pressing W or S combined with a strafe key causes the player to receive more velocity than simply holding W or S. Combining this with turning the player's camera in line with the strafe key, a player can get lots of momentum to jump to far platforms. This is normally called a **prestrafe**. This can be further improved by letting go of W in the air but continuing the mouse and strafe key pattern, called **airstrafing**. gaining speed before the jump and while in the air lets you travel futher. This is called a **long jump**. There are 2 other forms of longjump in 1.6 that are more advanced, [countjumps](https://kz-rush.com/en/page/countjump-physics) and [highjumps](https://kz-rush.com/page/highjump-physics). 
```

Comment:
> Strafing works in the same way that bunny hopping does; pressing W or S combined with a strafe key causes the player to receive more velocity than simply holding W or S. Combining this with turning the player's camera in line with the strafe key, a player can get lots of momentum to jump to far platforms. This is normally called a **prestrafe**.

strafing usually refers to airstrafing in quake-style movement games. perhaps better worded as something like this:

> Prestrafing works in the same way that airstrafing does; pressing a movement key and moving the mouse causes the player to receive more velocity. Most commonly a combination of forward and left or right keys are used for prestrafing while moving the mouse left or right accordingly.

perhaps move this to after the explanation of airstrafing.